bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -688,17 +690,17 @@ bool ReplicaImp::tryToSendPrePrepareMsg(bool batchingLogic) {
     auto builtReq = buildPrePrepareMessage();
     isSent = builtReq.second;
     if (isSent) {
-      pp = builtReq.first;
+      pp = std::move(builtReq.first);
       batch_closed_on_logic_off_++;
       time_to_collect_batch_ = MinTime;
     }
   }
   if (!pp) return isSent;
-  startConsensusProcess(pp);
+  startConsensusProcess(pp.release());
```

Comment:
so you are releasing here.
Is there an existing task to conmtinue inside startConsensusProcess? I might be repeating/missing info which you told me. But i prefer doing so than miss an important task..
is it https://jira.eng.vmware.com/browse/BC-26194?